es/package.json

```
@@ -0,0 +1,34 @@
+{
+  "name": "cybozu-protobuf-es",
+  "private": "true",
+  "scripts": {
+    "build": "tsc -b tsconfig.build.json",
+    "clean": "rimraf \"packages/**/lib\" \"packages/**/*.tsbuildinfo\"",
+    "fix": "npm run fix:eslint && npm run fix:prettier",
+    "fix:eslint": "npm run lint:eslint -- --fix",
+    "fix:prettier": "npm run lint:prettier -- --write",
+    "lint": "concurrently \"npm:lint:*\"",
+    "lint:eslint": "eslint --ext .ts ./packages/**/src",
+    "lint:prettier": "prettier --check ./packages/**/src",
+    "lint:tsc": "tsc -b tsconfig.build.json --noEmit"
+  },
+  "workspaces": [
+    "packages/*"
+  ],
+  "devDependencies": {
+    "@typescript-eslint/eslint-plugin": "5.56.0",
+    "@typescript-eslint/parser": "5.56.0",
+    "concurrently": "^7.6.0",
```

Maybe, you just forgot to delete the caret for this line. So, I'll fix this line.

```suggestion
    "concurrently": "7.6.0",
```